src/EditorFeatures/CSharpTest/Completion/CompletionProviders/XmlDocumentationCommentCompletionProviderTests.cs

```diff
@@ -760,6 +760,35 @@ static void Goo()
                 """, "cref", "langword", "href");
         }
 
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/72259")]
+        public async Task SeeAttributeNames2()
+        {
+            var text = """
+                class C
+                {
+                    /// <summary>
+                    /// <see l$$=""/>
```

Comment:
Please add test like `/// <see l$$     =""/>`. Since there is a gap between identifier and a `=` (even if there is at least one space, I made several just for clarity) I would expect completion to actually insert `=""`: `/// <see langword="$$"     =""/>`. This way user's path can be the following:
1. `/// <see l$$     =""/>`
2. `/// <see langword="$$"     =""/>`
3. `/// <see langword="not$$"     =""/>` - complete the keyword
4. `/// <see langword="not"     $$=""/>` - move cursor to the next "empty" attribute
5. `/// <see langword="not"     langword="$$"/>` - work on the next attribute

Reply:
I added a test with a space that adds the equals/quotes, but the other parts seemed a little out of scope for this issue

Reply:
I don't know what do you mean by "the other parts". I've written potential user path just to show that the behavior I expect here can actually be useful. And by looking at new tests I see that you fully fulfilled this request, so treat this comment as resolved, thanks for your work!